cni/pkg/install/binaries.go

```diff
@@ -22,10 +22,13 @@ import (
 )
 
 func copyBinaries(srcDir string, targetDirs []string, updateBinaries bool, skipBinaries []string) error {
-	skipBinariesSet := arrToSet(skipBinaries)
+	skipBinariesSet := make(map[string]struct{}, len(skipBinaries))
```

Comment:
can you use istio.io/istio/pkg/util/sets

Reply:
got it